demo/lib/vm.bash

```diff
@@ -143,25 +195,46 @@ vm-reboot() { # script API
 
 vm-networking() {
     vm-command-q "grep -q 1 /proc/sys/net/ipv4/ip_forward" || vm-command "sysctl -w net.ipv4.ip_forward=1"
-    vm-command-q "grep -q ^net.ipv4.ip_forward=1 /etc/sysctl.conf" || vm-command "sed -i 's/#net.ipv4.ip_forward=1/net.ipv4.ip_forward=1/g' /etc/sysctl.conf"
+    vm-command-q "grep -q ^net.ipv4.ip_forward=1 /etc/sysctl.conf" || vm-command "echo net.ipv4.ip_forward=1 > /etc/sysctl.conf"
```

Comment:
feeling a bit uneasy on replacing sysctl.conf contents with this, yet it probably works for us in clean VMs... perhaps we could

`vm-sed-file /etc/sysctl.conf '/net.ipv4.ip_forward/d'`
and then append (`>> /etc/sysctl.conf`) here.

Reply:
> feeling a bit uneasy on replacing sysctl.conf contents with this, yet it probably works for us in clean VMs... perhaps we could
> 
> `vm-sed-file /etc/sysctl.conf '/net.ipv4.ip_forward/d'`
> and then append (`>> /etc/sysctl.conf`) here.

Ouch, well spotted. That was unintentional. It was supposed to be appending to and not nuking the distro-default sysctl.conf altogether. I guess `echo net.ipv4.ip_forward=1 > /etc/sysctl.d/98-ip-forward.conf` would be even better.

Reply:
Yep, that's nice.